cmd/nginx/flags.go

```diff
@@ -147,6 +149,8 @@ Requires the update-status parameter.`)
 			`Dynamically refresh backends on topology changes instead of reloading NGINX.
 Feature backed by OpenResty Lua libraries.`)
 
+		bindAddresses = flags.String("bind-addresses", "0.0.0.0,[::]",
```

Comment:
This should be a valid IPV6 address (remove the [] chars)

Reply:
No need to add "Default..." in the flag description. The flag package does it for you during compilation.